spec/requests/admin/ambassador_task_assignments_controller_spec.rb

```diff
@@ -0,0 +1,222 @@
+require "spec_helper"
+
+RSpec.describe Admin::AmbassadorTaskAssignmentsController, type: :request do
```

Comment:
One trend I've seen recently, with request specs, is making multiple requests in a single spec.

I've previously been somewhat skeptical, because it seems like it muddies up the test isolation - but having read a bunch more request specs, I'm not convinced it's actually terrible.

This isn't something you need to do here, just something I've been thinking about